File: scripts/http_client.rb
#! /usr/bin/env ruby

require 'net/http'
require 'uri'
require 'date'

ARGV.each do |url|
	begin
	  res = Net::HTTP.get_response(URI.parse(url))
		puts "HTTP/#{res.http_version} #{res.code} #{res.msg}"
		puts "Date: #{Time.now.strftime('%Y/%m/%d %H:%M:%S')}"
	  puts ""
	rescue => e
		puts e.message
	end
end

File: gomi_codes/http_client.go
package main

import (
	"flag"
	"fmt"
	"time"
	"net/http"
)

func main() {
	flag.Parse()

	res, err := http.Get(flag.Arg(0))
	if err != nil {
		fmt.Println(err)
		return
	}
	fmt.Printf("%v %v\n", res.Proto, res.Status)
	fmt.Printf("Date: %v\n", time.Now().Format("2006-01-02 15:04:05"))
}